src/test/java/org/prebid/server/auction/TargetingKeywordsCreatorTest.java

```diff
@@ -294,23 +294,29 @@ public void shouldIncludeBidderKeysTargetingIfIncludeBidderKeysFlagIsTrue() {
 
         // when
         final Map<String, String> keywords = TargetingKeywordsCreator.create((String) null, false, true, false)
-                .makeFor(bid, "bidder1", true, null, null, null, null, null);
+                .makeFor(bid, "bidder1", true, null, null, null, null, null, false);
 
         // then
         assertThat(keywords).containsKeys("hb_bidder_bidder1", "hb_pb_bidder1");
     }
 
     @Test
-    public void shouldIncludeWinUrlWhenWinUrlIsNotNullAndBidIsWinning() {
+    public void shouldIncludeWinUrlWhenBidIsNotVideoAndWinUrlIsNotNullAndBidIsWinning() {
```

Comment:
Do we need a test that verifies that `hb_winurl` and `hb_bidid` _are not set_ for video bids?

Reply:
Will do